.github/workflows/linters.yml

```diff
@@ -0,0 +1,56 @@
+name: Linters
+
+on: pull_request
+
+env:
+  FORCE_COLOR: 1
+
+jobs:
+  lighthouse:
+    name: Lighthouse
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Lighthouse
+        run: npm install -g @lhci/cli@0.7.x
+      - name: Lighthouse Report
+        run: lhci autorun --upload.target=temporary-public-storage --collect.staticDistDir=.
+  webhint:
+    name: Webhint
+    runs-on: ubuntu-18.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v1
+        with:
+          node-version: "12.x"
+      - name: Setup Webhint
+        run: |
+          npm install --save-dev hint@7.x
+          [ -f .hintrc ] || wget https://raw.githubusercontent.com/microverseinc/linters-config/master/html-css/.hintrc
+      - name: Webhint Report
+        run: npx hint .
+  stylelint:
+    name: Stylelint
```

Comment:
- [ ] [OPTIONAL] Please ensure your commit messages all begin with capital letters and it is written using the imperative mood. 
 Eg. 
-  _"Add linter files"_  instead of _"linters file added to the html-css-project"_ 
-  _"make xyzzy do frotz"_ instead of _"[This patch] makes xyzzy do frotz" or "[I] changed xyzzy to do frotz"_
[Read more](https://initialcommit.com/blog/git-commit-messages-best-practices)
Getting in the habit of creating quality commit messages makes using and collaborating with Git a lot easier.